job-server/src/spark.jobserver/AkkaClusterSupervisorActor.scala

```diff
@@ -205,10 +205,15 @@ class AkkaClusterSupervisorActor(daoActor: ActorRef) extends InstrumentedActor {
           return
       }
 
+    var driverMemory = "0"
+    if (contextConfig.hasPath("spark.driver") && contextConfig.hasPath("spark.driver.memory")) {
```

Comment:
Not enough, the `driver` section might not exist completely.
Similar check you have for `spark.proxy` in here https://github.com/spark-jobserver/spark-jobserver/pull/441/files#diff-8a327aff8a0090a45c45ca1544981684R216